OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -112,7 +112,9 @@ public override Activity Tick(Actor self)
 
 			// Determine where to search from and how far to search:
 			var searchFromLoc = GetSearchFromLocation(self);
-			var searchRadius = harv.LastOrderLocation.HasValue ? harvInfo.SearchFromOrderRadius : harvInfo.SearchFromProcRadius;
+			var fromOrderRadius = (self.Owner.IsBot || !self.Owner.Playable) ? harv.AIFromOrderScanRadius : harvInfo.SearchFromOrderRadius;
+			var fromRefineryRadius = self.Owner.IsBot || !self.Owner.Playable ? harv.AIFromRefineryScanRadius : harvInfo.SearchFromRefineryRadius;
```

Comment:
Above you have brackets around the condition, here not. Please chose one style for both.